README.md

```diff
@@ -8,16 +8,14 @@ It *stores all keys locally* in the browser, *never exposing your keys to anyone
 
 ## Getting started
 
-BitShares-UI depends node Node.js, and version 8+ is required.
+BitShares-UI depends node Node.js, we recommend using version 10 and above.
 
-On Ubuntu and OSX, the easiest way to install Node is to use the [Node Version Manager](https://github.com/creationix/nvm).
-
-To install NVM for Linux/OSX, simply copy paste the following in a terminal:
+To install NODE for Linux/Ubuntu 20.04.2 LTS, simply copy paste the following in a terminal:
 
 ```
-curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.30.2/install.sh | bash
-nvm install v9
-nvm use v9
```

Comment:
Why did you remove the usage of nvm?

Reply:
Perhaps it's not needed in the UI user setup procedure; switching between node versions or "Node Version Manager" is perhaps an extra admin server feature and not a requirement for setup the UI.

Reply:
@AmmarYousefM although I agree it's good to install nodejs via apt on Ubuntu, you removed instructions about macOS (formerly OSX), which is not very good (btw I didn't test whether the old instruction would still work on macOS).

Reply:
@abitmore will be testing the same commands over my OSX and get back to this PR with an update.

Reply:
I had tested the below procedure over OSX/macOS Catalina v10.15.7 (Latest) and it went successfully:

curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.30.2/install.sh | bash
nvm install v10
nvm use v10
git clone https://github.com/bitshares/bitshares-ui.git
cd bitshares-ui
npm install
npm run build

Note: since nvm can be generalized across OSX and Ubuntu, it's good to keep it in place rather than writing two procedures for every OS.